autoscheduler/frontend/src/components/LandingPage/LandingPage.css

```diff
@@ -6,15 +6,13 @@
 }
 
 .dialog-container {
-    width: 100%;
+    padding-top: 32px;
```

Comment:
Did you mean to add this in the latest commit? I don't remember us talking about changing this spacing, and both this and `dialog-link` have 32px top padding now which looks a bit odd to me. If it was intentional, you should only put the padding on one of the elements 

| Before | After |
|-|-|
| ![image](https://user-images.githubusercontent.com/28655462/111890590-ba72b700-89b8-11eb-87d0-1edc09551c8d.png) | ![image](https://user-images.githubusercontent.com/28655462/111890583-ae86f500-89b8-11eb-808b-87695f2c49a0.png) |

Reply:
Correct yeah I definitely didn't meant to do that lol, will undo